packages/lib/src/vis/heatmap/HeatmapVis.tsx

```diff
@@ -81,7 +88,12 @@ function HeatmapVis(props: Props) {
   const maskArray = useMask(dataArray, ignoreValue);
 
   return (
-    <figure className={styles.root} aria-label={title} data-keep-canvas-colors>
+    <figure
+      className={`${styles.root} ${className}`}
```

Comment:
Just concatenating the two classes by hand; I don't think it's worth an extra library.